nextflow.config

```diff
@@ -48,17 +45,6 @@ params {
 // Analysis params
   sample_vcfs = false
 
-// Nextstrain data
```

Comment:
Delete `test_nextstrain.config` as well

Reply:
Also should delete `nextstrain-augur` from `environment.yaml`, the `nextstrain_ch` from `main.nf`, and references to `test_nextstrain.config` in `nextflow.config`